### PR TITLE
adding new docs for dbt assist

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds new docs for dbt assist as reviewed/discussed with Greg and team. 

[notion project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb)

### CODE REVIEW

File: website/docs/docs/cloud/about-cloud/about-dbt-cloud.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ dbt Cloud's [flexible plans](https://www.getdbt.com/pricing/) and features make
 
 <Card
     title="dbt Cloud IDE"
-    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser."
+    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser. Use dbt Assist, a powerful AI feature that automatically generates tests and documentation."
```
Comment (mirnawong1) [inline website/docs/docs/cloud/about-cloud/about-dbt-cloud.md]:
```suggestion
    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser. Use dbt Assist, a powerful AI feature that automatically generates documentation, metrics, semantic models, and tests."
```

File: website/sidebars.js [hunk 2]
```diff
@@ -254,6 +254,17 @@ const sidebarSettings = {
               "docs/cloud/dbt-cloud-ide/lint-format",
             ],
           },
+          {
+            type: "category",
+            label: "dbt Assist",
```
Comment (mirnawong1) [inline website/sidebars.js]: FYI I'm thinking of now adding this back to live inside the dbt cloud ide folder. This is to also make way for an addl dev tool that should live under 'dev w dbt cloud'.

So am going to change this on Monday 🙏

File: website/docs/docs/cloud/dbt-assist.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+--- 
+title: "About dbt Assist" 
+sidebar_label: "About dbt Assist" 
+description: "dbt Assist’s powerful AI feature helps you deliver data that works." 
+---
+
+# About dbt Assist <Lifecycle status='beta'/> 
+
+dbt Assist is a powerful artificial intelligence (AI) feature that helps automate dbt development, allowing you to focus on delivering data that works. dbt Assist’s AI generates documentation, metrics, semantic models, and tests for your dbt SQL and Python models directly in the dbt Cloud IDE, with a click of a button.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/dbt-assist.md]:
```suggestion
dbt Assist is a powerful artificial intelligence (AI) feature that helps automate development in dbt Cloud, allowing you to focus on delivering data that works. dbt Assist’s AI generates documentation, metrics, semantic models, and tests for your dbt SQL and Python models directly in the dbt Cloud IDE, with a click of a button.
```

File: website/docs/docs/cloud/dbt-assist-data.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+<expandable alt_header="Is my data used by dbt Labs to train AI models?">
+  
+  No, dbt Labs does not use your prompts to train any internal dbt Labs AI models. It uses API calls to the AI vendor you provide, or, if you choose to use our AI provider, to our AI provider. dbt Labs is not a party to the agreement between you and your AI vendor, and cannot advise as to what rights and obligations it provides.
+
+</expandable>
+
+<expandable alt_header="Does dbt Labs share my personal data with third parties">
+
+  dbt Labs only shares client personal information as needed to perform the services, under client instructions, or for legal, tax or compliance reasons.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/dbt-assist-data.md]:
```suggestion
  dbt Labs only shares client personal information as needed to perform the services, under client instructions, or for legal, tax, or compliance reasons.
```

File: website/docs/docs/cloud/dbt-assist-data.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+</expandable>
+
+<expandable alt_header="Does dbt Assist store or use personal data?">
+
+  dbt Assist only stores ephemerally any data input by the user. The user clicks the AI assist button. Ephemeral storage cannot be accessed or used by dbt Labs employees, and ephemeral storage does not persist on dbt Labs systems.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/dbt-assist-data.md]: Feels like some context is missing from the second sentence here

File: website/docs/docs/cloud/dbt-assist-data.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+</expandable>
+
+<expandable alt_header="Does dbt Assist access my warehouse data?">
+
+  dbt Assist utilizes the user’s prompt to initiate a search via client’s API of the client’s model provided by its AI vendor and returns probable output. Such output must be double-checked by the user for completeness and accuracy. This does not involve the use of client’s warehouse data.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/dbt-assist-data.md]:
```suggestion
  dbt Assist utilizes the user’s prompt to initiate a search via APIs of the client’s model provided by its AI vendor and returns probable output. Such output must be double-checked by the user for completeness and accuracy. This does not involve the use of the client’s warehouse data.
```

File: website/docs/docs/cloud/dbt-assist-data.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+<expandable alt_header="Does dbt Assist access my warehouse data?">
+
+  dbt Assist utilizes the user’s prompt to initiate a search via APIs of the client’s model provided by its AI vendor and returns probable output. Such output must be double-checked by the user for completeness and accuracy. This does not involve the use of the client’s warehouse data.
+
+</expandable>
+
+<expandable alt_header="Can dbt Assist data be deleted upon client written request?">
+
+  dbt Assist data, aside from usage data, does not persist on dbt Labs systems. Therefore, dbt Labs should not have possession of any data requiring deletion. Usage data is retained by dbt Labs, but users are advised to not include personal or sensitive information in the prompts. To the extent data is used to train AI models, that would be governed by the agreement between client and its AI Vendor, and dbt Labs is not a party to that agreement.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/dbt-assist-data.md]:
```suggestion
  dbt Assist data, aside from usage data, does not persist on dbt Labs systems. Therefore, dbt Labs does not have possession of any data requiring deletion. Usage data is retained by dbt Labs, but users are advised not to include personal or sensitive information in the prompts. To the extent data is used to train AI models, that would be governed by the agreement between the client and its AI Vendor, and dbt Labs is not a party to that agreement.
```

**Comment by mirnawong1** [comment]
Hey @matthewshaver some if the language here was from legal so will review on Monday to make sure it makes sense with what they said 🙏

File: website/docs/docs/cloud/about-cloud/about-dbt-cloud.md [hunk 8]
```diff
@@ -24,7 +24,7 @@ dbt Cloud's [flexible plans](https://www.getdbt.com/pricing/) and features make
 
 <Card
     title="dbt Cloud IDE"
-    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser."
+    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser. Use dbt Assist, a powerful AI feature that automatically generates documentation, metrics, semantic models, and tests."
```
Comment (mirnawong1) [inline website/docs/docs/cloud/about-cloud/about-dbt-cloud.md]:
```suggestion
    body="The IDE is the easiest and most efficient way to develop dbt models, allowing you to build, test, run, and version control your dbt projects directly from your browser. Use dbt Assist, a powerful AI co-pilot feature that automatically generates documentation, metrics, semantic models, and tests."
```

**Pull request merged**